### PR TITLE
productlisting logic

### PR DESCRIPTION
**Pull request by marcushohlbein**

### CODE REVIEW

File: client/src/components/ProductList/ProductList.js [hunk 1]
```diff
@@ -4,7 +4,30 @@ import styled from 'styled-components/macro'
...
-  const { styleId, media, brand, model, retailPrice, sale } = props.product
+  const { styleId, media, brand, model, shops, sale } = props.product
+
+  function shopCount(count) {
+    if (count > 1) {
+      return count + ' Shops'
+    } else {
+      return count + ' Shop'
+    }
+  }
```
Comment (macfrei) [inline client/src/components/ProductList/ProductList.js]: Just another way to write it :)
```suggestion
  const shopCount = count => count > 1 ? count + ' Shops' : count + ' Shop'
```

**Review by macfrei — COMMENTED** [review]
Well done! 🚀

File: client/src/components/ProductList/ProductList.js [hunk 2]
```diff
@@ -4,7 +4,30 @@ import styled from 'styled-components/macro'
...
+      .sort((a, b) => a > b)
+    const salesPrice = shops
+      .map(product => product.salesPrice)
+      .sort((a, b) => a > b)
+    if (salesPrice[0] !== '' && regularPrice[0] > salesPrice[0]) {
+      return salesPrice[0]
+    } else {
+      return regularPrice[0]
+    }
+  }
```
Comment (bwnstck) [inline client/src/components/ProductList/ProductList.js]:
```suggestion
    if (salesPrice[0] !== '' && regularPrice[0] > salesPrice[0]) return salesPrice[0]
    return regularPrice[0]
    }
```
This should work as well, and its easier to read ... 

Look at this https://www.youtube.com/watch?v=EumXak7TyQ0

**Review by bwnstck — APPROVED** [review]
🚀 👍

File: client/src/components/ProductList/ProductList.js (hunk 2 above)
Comment (marcushohlbein) [reply client/src/components/ProductList/ProductList.js]: great. thanks for the advise 🤙

**Pull request merged**